chat-plugins/uno.js

```diff
@@ -0,0 +1,546 @@
+'use strict';
+
+const maxTime = 20; // seconds
+
+function cardHTML(card, fullsize) {
+	let surface = card.value.replace(/[^A-Z0-9\+]/g, "");
+	return `<button style="height: 140px; padding-bottom: 120px; text-align: left; font-weight: bold; border: 2px solid rgba(33 , 68 , 72 , 0.59); min-width: ${fullsize ? "80" : "30"}px; border-radius: 10px 2px 2px 3px; color: ${card.colour === "Black" ? "yellow" : "black"}; background-color: ${card.colour}" name=send value="/uno play ${card.name}">${surface}</button>`;
+}
+
+function createDeck() {
+	let colours = ["Red", "Blue", "Green", "Yellow"];
+	let values = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "Reverse", "Skip", "+2"];
+
+	let basic = [];
+
+	for (let i = 0; i < 4; i++) {
+		let colour = colours[i];
+		basic.push(...values.map(v => {
+			return {value: v, colour: colour, name: colour + " " + v};
+		}));
+	}
+
+	return [...basic, ...basic, // two copies of the basic stuff (total 96)
+		...[0, 1, 2, 3].map(v => ({colour: colours[v], value: "0", name: colours[v] + " 0"})), // the 4 0s
+		...[0, 1, 2, 3].map(v => ({colour: "Black", value: "Wild", name: "Wild"})), // wild cards
+		...[0, 1, 2, 3].fill(0).map(v => ({colour: "Black", value: "+4", name: "Wild +4"})), // wild +4 cards
```

Comment:
ah my bad, I forgot to remove that when i was switching from your method to Zarel's